crates/flow_cli_common/src/lib.rs

```diff
@@ -0,0 +1,31 @@
+//! Contains helpers and things that are used by all Flow rust executables.
+
+mod logging;
+
+pub use logging::{init_logging, LogArgs, LogFormat};
+
+/// The flowctl CLI boilerplate requires that all subcommands have a type for thier arguments, so
+/// this is a type to use with subcommands that accept no arguments.
+#[derive(Debug, clap::Args)]
+pub struct NoArgs;
+
+/// Helper trait for exiting the application early if there's an error.
+pub trait OrBail<T> {
+    fn or_bail(self, message: &str) -> T;
+}
```

Comment:
TBH, I'm thinking we should discourage its use in favor of just returning a `Result<(), anyhow::Error>`. I figured I'd keep it around temporarily in case any of the other CLIs use it. I think I'll be able to delete it pretty quickly, though.